requirements/base.in

```diff
@@ -20,6 +21,7 @@ django-model-utils
 django-allauth
 django-celery-beat
 django-crispy-forms
+django-phonenumber-field[phonenumberslite]
```

Comment:
Is it necessary to have this extra dependency? This data comes straight from CCID with no way to edit it in CCC so it doesn't seem that useful to have the extra validation and support.

Reply:
Seems reasonable to me. I can remove it.